Sources/Brave/Frontend/Browser/Playlist/BasicAuthCredentials.swift

```diff
@@ -5,44 +5,87 @@
 
 import Foundation
 import UIKit
+import BraveCore
+import os.log
 
 class BasicAuthCredentialsManager: NSObject, URLSessionDataDelegate {
-  private let domains: [String]
+  private static var credentials = [String: URLCredential]()  // ["origin:port": credential]
   
-  init(for domains: [String]) {
-    self.domains = domains
+  public static let validDomains: Set<String> = [
+    // Brave Search
+    "search.brave.com", "search.brave.software",
```

Comment:
Do we need a basic auth for search.brave.com?

Reply:
No I don't think we do. I can remove it. The old code had it, so I copied that over. But I just checked and it isn't needed :)